ai-service/db.py

```
@@ -0,0 +1,36 @@
+import numpy as np
+if not hasattr(np, 'float_'):
+    np.float_ = np.float64
+
+import uuid
+import chromadb
+
+# Set up ChromaDB client and persistent collection
+client = chromadb.PersistentClient(path="./chroma_store")
+collection = client.get_or_create_collection("code_chunks")
+
+def add_chunks(chunks: list[str], embeddings: list[list[float]]):
```

Ruff (0.12.2)

Missing return type annotation for public function `add_chunks`

Add return type annotation: `None`

(ANN201)